File: abc/abc102/a.cpp
// https://atcoder.jp/contests/abc102/submissions/26576772

#include <bits/stdc++.h>
using namespace std;
int main() {
  int n, ans = 0;
  cin >> n;
  for(int i = n; ans == 0; i++) if(i % 2 == 0 && i % n == 0) ans = i;
  cout << ans;
}

File: abc/abc102/a.java
// https://atcoder.jp/contests/abc102/submissions/43810436

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();

    if(n % 2 == 0) {
      System.out.println(n);
    } else {
      System.out.println(n * 2);
    }
  }
}
